src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/BodyModelBinder.cs

```diff
@@ -192,13 +192,10 @@ public async Task BindModelAsync(ModelBindingContext bindingContext)
 
         private bool ShouldHandleException(IInputFormatter formatter)
         {
-            var policy = _options.InputFormatterExceptionPolicy;
-
-            // Any explicit policy on the formatters takes precedence over the global policy on MvcOptions
-            if (formatter is IInputFormatterExceptionPolicy exceptionPolicy)
-            {
-                policy = exceptionPolicy.ExceptionPolicy;
-            }
+            // Any explicit policy on the formatters overrides the default.
+            var policy = (formatter is IInputFormatterExceptionPolicy exceptionPolicy) ?
```

Comment:
could be simplified to an `(x as Y)?.Z ?? blah;`

Reply:
Seems similar to me but :ok: